pkg/executables/kubectl.go

```diff
@@ -339,21 +339,27 @@ func (k *Kubectl) ValidateControlPlaneNodes(ctx context.Context, cluster *types.
 
 func (k *Kubectl) ValidateWorkerNodes(ctx context.Context, cluster *types.Cluster, clusterName string) error {
 	logger.V(6).Info("waiting for nodes", "cluster", clusterName)
-	md, err := k.GetMachineDeployment(ctx, cluster, clusterName, WithCluster(cluster), WithNamespace(constants.EksaSystemNamespace))
+	deployments, err := k.GetMachineDeployments(ctx, WithCluster(cluster), WithNamespace(constants.EksaSystemNamespace))
 	if err != nil {
 		return err
 	}
+	for _, deployment := range deployments {
+		md, err := k.GetMachineDeployment(ctx, cluster, deployment.Name, WithCluster(cluster), WithNamespace(constants.EksaSystemNamespace))
+		if err != nil {
+			return err
+		}
 
-	if md.Status.Phase != "Running" {
-		return fmt.Errorf("machine deployment is in %s phase", md.Status.Phase)
-	}
+		if md.Status.Phase != "Running" {
```

Comment:
Is this because these are tabs instead of spaces? I wonder why it's highlighting like that